docs/add-ons/testing.md

```diff
@@ -62,7 +62,7 @@ docker build \
 
 If you don't want to use the devcontainer environment, you can still run add-ons locally with Docker.
 
-Create a new folder for data and add a test _options.json_ file. After that you can run your add-on with:
+Create a new folder for data and add a test `options.json` file. After that you can run your add-on with:
```

Comment:
Shouldn't this be the `options.yaml` file to be consistent with the rest of the documentation nowadays?

Reply:
No, this is the file that is written by the supervisor inside the add-on
We should replace this with `bashio`

Reply:
Good morning @frenck... 🤦

Reply:
> We should replace this with `bashio`

Isn't the `options.json` file required in every case anyway? From my understanding, `bashio` could parse the file, but it still needs to be present, doesn't it?

Reply:
`bashio` doesn't use the file anymore :)
It queries the Supervisor API instead.

Reply:
🤯🔥